### PR TITLE
'Archival Tape' Announcer

### PR DESCRIPTION
**Pull request by francinum**
## About The Pull Request

So I noticed that bay had pretty much every asset required for this.

## Why It's Good For The Game

More variation. It's funny.

## Changelog
:cl:
soundadd: New Station Announcer trait voice pack
/:cl:

### CODE REVIEW

**Comment by Kapu1178** [comment]
baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed baysed

File: code/datums/station_traits/neutral_traits.dm [hunk 1]
```diff
@@ -54,8 +56,29 @@
...
+		/datum/station_trait/announcement_baystation
+		)
 
 /datum/station_trait/announcement_medbot/New()
 	. = ..()
 	SSstation.announcer = /datum/centcom_announcer/medbot
+
+/datum/station_trait/announcement_intern/New()
+	. = ..()
+	SSstation.announcer = /datum/centcom_announcer/intern
```
Comment (Sarchutar) [inline code/datums/station_traits/neutral_traits.dm]: This is already present and is a duplicate of a thing ~20 lines above

**Comment by Raven-Industries** [comment]
Phase 1 of Franc's secret plan to turn Bee into baycode piece by piece.

**Comment by Troontjelolo** [comment]
> Phase 1 of Franc's secret plan to turn Bee into baycode piece by piece.

franc_gates

**Review by Archanial — APPROVED** [review]
code is good

**Review by Crossedfall — APPROVED** [review]

**Pull request merged**